l3kernel/l3text.dtx

```diff
@@ -188,13 +188,12 @@
 %   \end{itemize}
 %
 %  Determining whether non-letter characters at the start of text should count
-%  as the uppercase element is controllable. When \cs{l_text_titlecase_check_letter_bool} is
-%  \texttt{true}, characters which are not letters (category code~$11$) are
+%  as the uppercase element is controllable. When
+%  \cs{l_text_titlecase_check_letter_bool} is \texttt{true}, codepoints which are
+%  not letters (Unicode general category \texttt{L}) are
 %  \enquote{skipped}: the first \emph{letter} is uppercased.
```

Comment:
not changed and only the first \emph{letter} found is uppercased.

Reply:
Changed